samples/client/petstore/python-tornado/petstore_api/models/additional_properties_class.py

```diff
@@ -63,7 +63,7 @@ class AdditionalPropertiesClass(object):
     def __init__(self, map_string=None, map_number=None, map_integer=None, map_boolean=None, map_array_integer=None, map_array_anytype=None, map_map_string=None, map_map_anytype=None, anytype_1=None, anytype_2=None, anytype_3=None, local_vars_configuration=None):  # noqa: E501
         """AdditionalPropertiesClass - a model defined in OpenAPI"""  # noqa: E501
         if local_vars_configuration is None:
-            local_vars_configuration = Configuration()
+            local_vars_configuration = Configuration.get_default_copy()
```

Comment:
Why not pass in local_vars_configuration=Configuration.get_default_copy()?

Reply:
Because the generated code leaves no place to manually instantiate the classes--they are instantiated as part of data returned from API calls.
Using the example in the related issue, I only instantiate the generated ApiClient, and MembersApi classes, passing in the configuration object as designed to ApiClient, then setting options on the instantiated objects, but the only way to toggle the "client_side_validation" value was to force the use of the stored default by means of this patch.